decentraland/unity-renderer
# Fix: new passport text refinement

## What does this PR change?

Fix #3983
Refinement of some new passport texts

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/passport-text-improvements
2. Verify all text changes listed in issue #3983

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
